src/Modules/DynamicContent/Livewire/CollectionEditor.php

```
@@ -78,10 +78,12 @@ public function save(): void
 
         if ( null === $this->editingRecordId ) {
             Gate::authorize( 'create', DynamicContentRecord::class );
+            // phpcs:ignore ArtisanPackUI.Security.ValidatedSanitizedInput -- $data is built from validated Livewire server state and persisted via Eloquent create() behind a Gate::authorize('create', ...) check.
             $manager->create( $type, $data );
         } else {
             $record = DynamicContentRecord::findOrFail( $this->editingRecordId );
```

PHPStan (2.2.7)

[error] Call to an undefined static method ArtisanPackUI\CMSFramework\Modules\DynamicContent\Models\DynamicContentRecord::findOrFail().


(staticMethod.notFound)